### PR TITLE
[wip]Add elementwise pow op

### PR DESCRIPTION
**Pull request by jacquesqiao**
project: https://github.com/PaddlePaddle/Paddle/issues/7769

### CODE REVIEW

**Comment by qingqing01** [comment]
There is already a Pow Op implementation in `operators/activation_op.h` :  https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/activation_op.h#L788

**Comment by jacquesqiao** [comment]
@qingqing01 yes, but the implementation in activation use power exponent as attribute, in leanring_rate_decay, the power exponent will be changed during training, it should be a Variable, I think in the future, the pow inactivation should also use this pow op.

File: python/paddle/v2/fluid/tests/test_elementwise_add_op.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def setUp(self):
         }
 
 
-class TestElementwiseAddOp_rowwise_add_0(TestElementwiseOp):
+class TestElementwiseAddAddOp_rowwise_add_0(TestElementwiseAddOp):
```
Comment (reyoung) [inline python/paddle/v2/fluid/tests/test_elementwise_add_op.py]: Why change this file?

**Comment by jacquesqiao** [comment]
@reyoung  sorry it's a typo, please review https://github.com/PaddlePaddle/Paddle/pull/7892 first. I will add gradient op for elementwise pow in this PR.

**Pull request closed without merging**